python/az/aro/azext_aro/custom.py

```diff
@@ -4,7 +4,7 @@
 import random
 import os
 
-import azext_aro.vendored_sdks.azure.mgmt.redhatopenshift.v2020_04_30.models as v2020_04_30
+import azext_aro.vendored_sdks.azure.mgmt.redhatopenshift.v2020_12_31_preview.models as v2020_04_30
```

Comment:
I don't know if this will break nightly prod e2es or not.
It will complicate keeping upstream az and local az in sync, but that is probably a step we will need to take at some point.
I don't think importing vfoo as vbar is a good idea.

Reply:
@mjudeikis @petrkotas we should catch up on what we want to do about this comment.

Reply:
This change should not be here at this point. Plan is to make this change once we refactor all code to stop using az and use SDK. This should introduce just xmsenum to new API